File: Mts.Core/Dto/ApiResponse.cs
﻿using System;
using System.Collections.Generic;
using System.Text;

namespace Mts.Core.Dto
{
    public class ApiResponse<T>
    {
        public ApiResponse()
        {
            Success = true;
            ErrorMesssage = new List<string>();
        }
        public bool Success { get; set; }
        public List<string> ErrorMesssage { get; set; }
        public T DataResponse { get; set; }
    }
}

File: Web/Mts.Web/ClientApp/app/dto/ApiResponse.ts
export class ApiResponse {
    public success: boolean;
    public errorMesssage: string[];
}

export class LoginApiResponse {
    public success: boolean;
    public errorMessage: string[];
    public dataResponse: AuthToken;
}

export class AuthToken {
    public token: string;
    public refreshToken: string;
}